b2b-sample/libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/identityProvider.ts

```diff
@@ -17,23 +17,6 @@
  */
 import { ControllerCallReturn, ControllerDecodeReturn } from "@b2bsample/shared/data-access/data-access-common-models-util";
```

Comment:
Add a new line after the license header

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c9fb90d7d7ab19441f2e0d135cf2a6f218b7c78b